### ISSUE TITLE
[BUG] - Hide the "** no tags **" group when there are no projects without tags

### ISSUE THREAD

**Issue by bediu** [alefragnani/vscode-project-manager#561]
The group "** no tags **" in the sidebar shows when it's empty as well. Seems unnecessary.

**Comment by alefragnani** [comment]
Hi @bediu ,

This is intentional. But yes, I could create a new setting to hide that, if desired.

Thank you

**Comment by bediu** [comment]
@alefragnani  Yes please, maybe even hide it by default unless an uncategorized project exists?

**Comment by ledleds** [comment]
This would be great! 👍

**Comment by CanePlayz** [comment]
Yup, I'd also love this

**Comment by alefragnani** [comment]
Thinking a bit more, there is no need for a setting. The new behavior will be _hide if empty_.

A contributed PR from @AlanQtten was merged and it should be available in the next release.

Stay tune

**Comment by CanePlayz** [comment]
Thanks!
